fg-stitch-lib/src/align/aligners/mod.rs

```diff
@@ -667,48 +668,110 @@ impl<'a, F: MatchFunc> SamRecordFormatter<'a, F> {
             return Ok([record].to_vec());
         }
 
+        // A note on how SAM flags are set
```

Comment:
while redundant (and mostly copied verbatim in the README), it is useful to detail here again for the developer.  The only worry is maintaining the same information in two places.  would we prefer to just link to the README here instead?